lib/chromic_pdf.ex

```diff
@@ -278,6 +278,26 @@ defmodule ChromicPDF do
         [chrome_args: "--font-render-hinting=none"]
       end
 
+  In some cases, chromic_pdf's default args may conflict with the ones you would like to add.
+
+  E.g. `--disable-gpu`
+
+  The problem is that you can't simply override them like so:
+
+  E.g. in default args within chromic_pdf's code, `--disable-gpu` and then
+  passing `chrome_args: "--enable-gpu"` won't work.
+
+  In this case, use Keyword form of the `:chrome_args` option which
+  allows targeted removing of problematic default args so that they don't
+  disrupt your chrome configuration.
+
+      defp chromic_pdf_opts do
+        [chrome_args: [append: "--headless=new --angle=swiftshader",
+                       remove: ["--headless", "--disable-gpu"]]]
+      end
+
+  _Note: `:append` expects a string whereas `:remove` expects a list._
```

Comment:
Let's be liberal in what the API can accept; a `List.wrap/1` somewhere will allow you to accept strings or lists for either of the options, and then we don't need this note in the docs.

```suggestion
```